src/main/database_manager.cpp

```diff
@@ -40,7 +40,9 @@ optional_ptr<AttachedDatabase> DatabaseManager::AttachDatabase(ClientContext &co
 	auto &db = DatabaseInstance::GetDatabase(context);
 	auto attached_db = db.CreateAttachedDatabase(info, db_type, access_mode);
 
-	InsertDatabasePath(info.path, attached_db->name);
+	if (db_type.empty()) {
```

Comment:
I get it, and saw it also elsewhere, but potentially could it make so that db_type is either some explicit string ("duckdb") or at least there is a function in the middle that allows to be a bit more clear like:
```
bool IsDuckDB(const string &type) {
    //FIXME!!
    return type.empty();
}
```

Reply:
Sure, we could do that in a separate PR